palisade-service/src/unit-tests/java/uk/gov/gchq/palisade/service/palisade/domain/DataRequestTest.java

```diff
@@ -63,9 +61,9 @@ public void storeAndRetrieveTest() {
         this.dataRequestRepository.save(entity);
         final DataRequestEntity subject = this.dataRequestRepository.getByRequestId("identifier-x");
 
-        assertThat("The user id is not preserved through persistence", subject.getUser().getUserId().getId(), is(equalTo("archibald")));
-        assertThat("The request id is not preserved through persistence", subject.getRequestId(), is(equalTo("identifier-x")));
-        assertThat("The context is not preserved through persistence", subject.getContext().get("testing repo"), is(equalTo("this")));
+        assertThat(subject.getUser().getUserId().getId()).isEqualTo("archibald");
+        assertThat(subject.getRequestId()).isEqualTo("identifier-x");
+        assertThat(subject.getContext().get("testing repo")).isEqualTo("this");
```

Comment:
Worth the change to use grouped assertions
```suggestion
        assertAll(
                () -> assertThat(subject.getUser().getUserId().getId()).isEqualTo("archibald"),
                () -> assertThat(subject.getRequestId()).isEqualTo("identifier-x"),
                () -> assertThat(subject.getContext().get("testing repo")).isEqualTo("this")
        );
```

Reply:
Could also try SoftAssertions:
```
SoftAssertions softly = new SoftAssertions();
softly.assertThat(.....
softly.assertThat(.....
softly.assertAll();
```

Reply:
The `assertSoftly` static method takes care of calling `assertAll()` before exiting.
```
@Test
void assertSoftly_example() {
  SoftAssertions.assertSoftly(softly -> {
    softly.assertThat("George Martin").as("great authors").isEqualTo("JRR Tolkien");
    softly.assertThat(42).as("response to Everything").isGreaterThan(100);
    softly.assertThat("Gandalf").isEqualTo("Sauron");
    // no need to call assertAll(), assertSoftly does it for us.
  });
}
```
Makes it a bit easier to read imo.